src/services/get-priority-options/get-priority-options.service.js

```diff
@@ -0,0 +1,9 @@
+const GetPriorityOptions = () => {
+  return [
+    { value: "low", label: "Low" },
+    { value: "medium", label: "Medium" },
+    { value: "high", label: "High" },
+  ];
+};
+
+export default GetPriorityOptions;
```

Comment:
There are no written rules about this, but the convention is to use snakeCase instead of PascalCase for methods... the latter is normally used to name components

Reply:
fce1599fba861da3c0a7cff34454d7be00ba2a38 Changed to camelCase as advised with service.